### ISSUE TITLE
Usability - Context menu items have a <button> tag inside them

### ISSUE THREAD

**Issue by chataylor** [OfficeDev/office-ui-fabric-react#2016]
### Bug Report

Package version(s): <TBD>
Browser and OS versions: All

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? No

Requested priority: Low

Products/sites affected: VSTS

#### Current Behavior: 
Using CAPS + Arrow keys, narrator is reading all the information as defined in programmatic UI. Hence, the screen reader users may find very annoying due to this multiple definition of the elements and where there is no such action performed by the interactive elements while navigating.

#### Expected Behavior: 
So, from the hierarchy, inappropriate "Button" definition should be removed.

#### User Impact:
Users relying on screen readers will get confused when narrator reading it as "Button"  and "Link" where there is no such action performed by the interactive elements while navigating with the keyboard. Inappropriate information will be delivered to the users.
